dlrover/python/master/hyperparams/simple_strategy_generator.py

```diff
@@ -73,11 +76,11 @@ def generate_opt_strategy(
             logger.info(f"paral_configs: {paral_configs}")
             return paral_configs[0]
 
-    def _generate_dataloader_config(
```

Comment:
Do not merge the methods to generate the dataloader and optimizer config. Otherwise, the method is too long.